tests/server/query_params/test_sort.py

```diff
@@ -98,3 +100,73 @@ def test_datetime_desc(get_good_response, structures):
         struct.get("attributes", {}).get("last_modified") for struct in response["data"]
     ]
     assert last_modified_list == expected_last_modified
+
+
+def test_unknown_field_errors(check_error_response, structures):
```

Comment:
```suggestion
def test_unknown_field_errors(check_error_response):
```

Reply:
Surprised flake8 let me get away with this...